setup.py

```diff
@@ -1,6 +1,16 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+
 from setuptools import setup
 
 
+def test_suite():
```

Comment:
to be honest it has been several years since I've just used `unittest`.

reading your comments I thought what you really wanted was to `python setup.py test` to work (I've seen some people do this in other repos) so after going through github and stackoverflow I learned about this and decided to add. :)